client/app/services/KeyboardShortcuts.js

```diff
@@ -35,11 +35,17 @@ const KeyboardShortcuts = {
 
   bind: keymap => {
     each(keymap, (fn, key) => {
-      const keys = key
+      // Resolve platform-specific modifiers and remove duplicates.
+      const rawKeys = key
         .toLowerCase()
         .split(",")
         .map(trim);
-      each(keys, k => {
+      // Translate platform-specific modifiers and dedupe.
+      const transformed = rawKeys.map(k =>
+        k.replace(/mod/g, modKey.toLowerCase())
```

Comment:
I'm just feeling that it might be better not to depends on "modKey", as "modKey" is not for mapping but for human readable name. (For example, the name might be localized in the future.)

How about just writing directly like below ?
```
   k.replace(/mod/g, (/Mac|iPod|iPhone|iPad/.test(navigator.platform) ? "command" : "ctrl") )
```